rs/crypto/internal/crypto_lib/threshold_sig/tecdsa/src/utils/group.rs

```diff
@@ -941,14 +941,14 @@ impl EccPoint {
 
     /// Constant-time multiscalar multiplication using Pippenger's algorithm
     ///
-    /// Return point_scalar_pairs[1].0 * point_scalar_pairs[1].1 + ...
-    /// + point_scalar_pairs[n].0 * point_scalar_pairs[n].1
+    /// Return
+    /// point_scalar_pairs[1].0 * point_scalar_pairs[1].1 + point_scalar_pairs[n].0 * point_scalar_pairs[n].1
```

Comment:
This seems wrong, removes the `...` to indicate a range

Reply:
(Also the starting index should be 0! But that was a bug in the original comment)

Reply:
```suggestion
    /// point_scalar_pairs[0].0 * point_scalar_pairs[1].1 + ...
    /// \+ point_scalar_pairs[n].0 * point_scalar_pairs[n].1
```

Does this look better?

Reply:
The escape `\` is needed, otherwise the MD generator will interpret the `+` as a bullet point.

Reply:
LGTM thank you!